### ISSUE TITLE
'Invalid appsecret_proof provided in the API argument' on dev and test apps

### ISSUE THREAD

**Issue by krb19xx** [SammyK/LaravelFacebookSdk#186]
Hello.

I have three Facebook Apps defined that are all really the same App - there's the Production instance and then a Development instance and a Test instance.

The Production instance works perfectly - all my API calls work as expected. On the Development and Test instances, NONE of my API calls work - they ALWAYS throw the error:
'Invalid appsecret_proof provided in the API argument'

I've tried EVERYTHING to get to the bottom of this, but I just can't figure it out. The code that's running on all 3 instances is EXACTLY the same (except for the APP_IDs and APP_SECRETs). I got the APP_IDs and APP_SECRETs from the Facebook Dashboard.

Basically, I have three versions of the same App, one of them is working perfectly and the other two are taking the appsecret_proof error.

Any help or ideas would be greatly appreciated.

**Comment by xevsy** [comment]
Have you fixed your issue? Because I have the same issue!

**Comment by yurik94** [comment]
same here, can't pass appsecret_proof to the request 👍
